javascript/javascript2/week3/fastestpresser.js

```diff
@@ -0,0 +1,56 @@
+// counting for L
+let totalL = 0;
+const keyL = document.querySelector("#countL");
+const pressL = (event) => {
+    if (event.keyCode == 76) {
+        totalL++;
+        keyL.innerHTML = totalL;
+    }
+};
+
```

Comment:
you should count how many times button is pressed but if I hold down button it is counting many presses

Reply:
updated "keydown" event with "keyup" event .Thankyou